File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;
import java.util.Map;

public interface BrandService {
    public List<TbBrand> findAll();
    public PageResult findByPage(int pageNum, int pageSize);
    public Result save(TbBrand tbBrand);
    public TbBrand findById(Long id);
    public Result update(TbBrand tbBrand);
    public Result deleteIds(Long [] ids);
    public List<Map> findList();
    public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);
}

File: pinyougou-parent/pinyougou-manager-web-9101/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findPage=function (pageNum,pageSize) {
        return $http.get('../brand/findByPage/'+pageNum+'/'+pageSize+'.do');
    }
    this.save=function (entity) {
        return $http.post("../brand/save.do",entity);
    }
    this.update=function(entity){
        return $http.post("../brand/update.do",entity);
    }
    this.delete=function (selectedIds) {
        return $http.post("../brand/deleteIds.do",selectedIds);
    }
    this.findById=function (id) {
        return $http.get("../brand/findById/"+id+".do");
    }
    this.findList=function(){
        return $http.get("../brand/findList.do");
    }
    this.findPage=function (page,rows,searchEntity) {
        return $http.post("../brand/search.do?page="+page+"&rows="+rows,searchEntity);
    }
});